tutorial/chapter/8/index.md

```diff
@@ -28,38 +30,39 @@ This will fail because `Array.prototype.map` actually passes *three* parameters
 
 ```javascript
 var xs = [1, 2, 3].map(function(data) {
-    return increment(data);
+    return helpers.add(data);
 });
 console.log(xs);
 ```
 
 ## Mutability problems
 
-Sometimes you need mutable data. Fortunately, it's still possible to create it
-in Squiggle. Squiggle does not expose a syntax like `x.y = z` to assign
-properties.
+Sometimes you need mutable data. Fortunately, it's still possible to create it in Squiggle. Squiggle does not expose a syntax like `x.y = z` to assign properties.
 
 ```squiggle
 let {Array, Object} = global
 let {assign} = Object
 let a = Array()
 let o = Object()
-let _ = assign(a, {0: "hi"})
-let _ = assign(o, {key: "value"})
-let _ = console.log([a, o]);
+assign(a, {0: "hi"})
+assign(o, {key: "value"})
+console.log([a, o])
 #=> [["hi"], {key: "value"}]
-in undefined
 ```
 
-These are the normal `Array` and `Object` functions from JavaScript, so they
-return unfrozen values.
+These are the normal `Array` and `Object` functions from JavaScript, so they return unfrozen values. There are plans to add mutable object literals and property assignment to Squiggle eventually.
 
 ## Problems with this or new
 
-Squiggle does not feature the keywords `this` or `new` from JavaScript because
-they cause more harm than good, and are not necessary (in general). Some
-libraries require their use, however, so Squiggle has functions for dealing with
-this.
+Squiggle does not feature the keywords `this` or `new` from JavaScript because they cause more harm than good, and are not necessary (in general). Some libraries require their use, however, so Squiggle has functions for dealing with this.
+
```

Comment:
I'm not sure what you mean. The following paragraphs show how to work with JS's `this` value. I don't have a strategy for `new` yet.